src/bluehawk/actions/ConsoleActionReporter.ts

```diff
@@ -12,9 +13,19 @@ import {
   StatesFoundEvent,
   WriteFailedEvent,
 } from "./ActionReporter";
+import * as path from "path";
+import chalk from "chalk";
+
+console.log(`
+${chalk.white.bgBlue.bold(
+  `            
+  Bluehawk  
+            `
+)}`);
```

Comment:
Probably don't want to execute at file scope - this will print even for library users loading this file